Makefile

```diff
@@ -121,34 +121,45 @@ send-plonk_bls12_381-proof-loop: ## Send a PLONK BLS12_381 proof using the task
 	@echo "Sending PLONK BLS12_381 proof in a loop every 10 seconds..."
 	@go run task_sender/cmd/main.go loop-tasks \
 		--proving-system plonk_bls12_381 \
-		--proof task_sender/test_examples/bls12_381/plonk.proof \
-		--public-input task_sender/test_examples/bls12_381/plonk_pub_input.pub \
-		--verification-key task_sender/test_examples/bls12_381/plonk.vk \
+		--proof task_sender/test_examples/gnark_plonk_bls12_381_script/plonk.proof \
+		--public-input task_sender/test_examples/gnark_plonk_bls12_381_script/plonk_pub_input.pub \
+		--verification-key task_sender/test_examples/gnark_plonk_bls12_381_script/plonk.vk \
 		--config config-files/config.yaml \
 		--interval 10 \
 		2>&1 | zap-pretty
 
+run-plonk_bls12_381-script: ## Run the gnark_plonk_bls12_381_script
```

Comment:
I think it is clearer about what it does if named something like
```
generate-plonk_bls12_381-proof
```

Same with the bn254 one